packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -200,14 +200,23 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
             "OptimismPortal: withdrawal hash has already been proven"
         );
 
+        // Compute the storage slot of the withdrawal hash in the L2ToL1MessagePasser contract.
```

Comment:
I could imagine in the future abstracting this behind a behind a function that pulls the version out of the withdrawal nonce and builds a particular input to `verifyInclusionProof`. Fine for now, but gives us a good upgrade path to verkle or mpt precompile